client/src/app/admin/panel/staff/admin-panel-departments.js

```diff
@@ -19,7 +20,18 @@ import SubmitButton from 'core-components/submit-button';
 import DropDown from 'core-components/drop-down';
 import Icon from 'core-components/icon';
 import Message from 'core-components/message';
+import Loading from 'core-components/loading'
 
+function getDefaultDepartmentList(){
+    return _.filter(SessionStore.getDepartments(), function(item) { return item.private != 1})
+}
+
+function getDepartmentIndexFromDepartmentId(departmentId) {
```

Comment:
`getDepartmentIndexFromDepartmentId` should indicate you're returing the index from `public` departments